resources/qml/Sidebar.qml

```diff
@@ -214,7 +214,7 @@ Rectangle
                 anchors.left: parent.left
                 anchors.leftMargin: model.index * (settingsModeSelection.width / 2)
                 anchors.verticalCenter: parent.verticalCenter
-                width: parent.width / 2
+                width: 0.5 * parent.width - (modesListModel.get(index).showFilterButton ? toggleFilterButton.width : 0)
```

Comment:
The ternary switch here is not for the width of the toggleFilterButton, but for whether the toggleFilterButton is shown for this mode, ie: 0 for "simple", toggleFilterButton.width for "advanced". The toggleFilterButton hides itself when the "simple" tab is showing, to not clutter up the simple mode.